config/config.go

```diff
@@ -14,6 +14,7 @@ type Config struct {
 	RouterURL                  string        `envconfig:"ROUTER_URL"`
 	DatasetControllerURL       string        `envconfig:"DATASET_CONTROLLER_URL"`
 	TableRendererURL           string        `envconfig:"TABLE_RENDERER_URL"`
+	UploadServiceURL           string        `envconfig:"UPLOAD_SERVICE_URL"`
```

Comment:
Do we need to proxy upload service calls to the API router? If we do, then we should not have this config (and use the API router URL instead)